src/Widgets/Modal.hs

```diff
@@ -57,6 +57,9 @@ createModal outsideStateE defaultState contentWidget size = fmap resultOrError .
                      ) $
          elClass "div" "modal-content" contentWidget
     performEvent_ $ changeState <$> stateEvs
+    when (defaultState == Active) $
+      -- TODO: why can't I replace the following line with: changeState Active
```

Comment:
That's because of rudimentary legacy from previous version of qua-view: modals are created at the program startup, and can be shown via bootstrap's js_showModal. Actually, it would be nice to change the modals to be regular reflex widgets

Reply:
n'ah, it complain about `Could not deduce: Performable m ~ m` ...

    Expected type: Event t (Performable m ())
    Actual type: Event t (m ())

but never mind, it's not much duplication...

Reply:
ah really? I thought in the end that didn't work out for some reason...?